app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,14 @@
 class Api::BaseController < ApplicationController
   layout nil
+
+  # Method to log login attempts
+  def log_login_attempt(user_id:, attempted_at:, successful:, ip_address:)
```

Comment:
Logging login attempts is a sensitive operation. Ensure that the 'LoginAttempt.log_attempt' method handles data securely, especially when logging IP addresses and user identifiers, to comply with privacy laws like GDPR.